feedback-module/src/components/Module.js

```diff
@@ -175,94 +175,105 @@ function Module({ pagetitle, endpoint, dir }) {
+                className="font-sans-md2 feedback-module__heading feedback-module__heading--inverse"
+                dangerouslySetInnerHTML={{ __html: t(screen.titleInverse) }}
+              ></p>
+            </Grid>
           )}
-          {screen.plainText &&
-            t(screen.plainText) &&
-            t(screen.plainText).map((paragraph, index) => {
-              return (
-                <p
-                  className="font-sans-md feedback-module__plaintext"
-                  key={index}
-                >
-                  {paragraph}
-                </p>
-              );
-            })}
-          <Form onSubmit={handleSubmit}>
-            {screen.checkboxes && t(screen.checkboxes.labels) && (
-              <>
-                {checkboxError && (
-                  <ErrorAlert errorText={t("errorMessages.checkboxError")} />
-                )}
-                <CheckboxList
-                  feedbackCheckboxes={t(screen.checkboxes.labels)}
-                  onCheck={(index) => onCheck(index)}
-                  setOtherField={setOtherField}
-                  checkboxKey={screen.checkboxes.labels}
-                  firstCheckRef={firstCheckRef}
-                  checkedFields={checkedFields}
-                  otherTooLong={otherTooLong}
-                  setOtherTooLong={setOtherTooLong}
-                />
-              </>
-            )}
-            {screen.textInputs && t(screen.textInputs) && (
-              <TextboxList
-                inputs={t(screen.textInputs)}
-                setInputQuestions={setInputQuestions}
-                inputQuestions={inputQuestions}
-                inputRefs={inputRefs}
-              />
+
+          <LightContainer formID={screen.formID}>
+            {loading && <LoadingSpinner overlay />}
+            {screen.title && (
+              <p
+                className="font-sans-md2 feedback-module__heading feedback-module__heading--default"
+                id="screenTitle"
+              >
+                {`${t(screen.title, { page: pagetitle })}${
+                  screen.checkboxes && screen.checkboxes.required ? "*" : ""
+                }`}
+              </p>
             )}
-            {screen.buttons &&
-              screen.buttons.map(
-                ({ type, text, nextScreen, feedbackID }, index) => {
-                  return (
-                    <ModuleButton
-                      buttonText={t(text)}
-                      className={`flex-button flex-button--${type}`}
-                      networkError={
-                        failedRequest && index === screen.buttons.length - 1
-                      }
-                      onClick={() =>
-                        handleClick(type, text, nextScreen, feedbackID)
-                      }
-                      key={index}
-                      type={type}
+            {screen.plainText &&
+              t(screen.plainText) &&
+              t(screen.plainText).map((paragraph, index) => {
```

Comment:
NOt a blocker, but I'm curious if you have added `lodash` in this project? feels like a really long null check!

Reply:
That's definitely something we were interested in looking into adding, but we never got the chance!